src/core/messageRouter.ts

```diff
@@ -0,0 +1,27 @@
+import TelegramMessage from '../classes/TelegramMessage';
+import UseCase from '../interfaces/useCase';
+import TelegramMessageType from '../classes/TelegramMessageType';
+
+export default class MessageRouter {
+  private useCases: UseCase[] = [];
+
+  registerUseCase(useCase: UseCase): void {
+    this.useCases.push(useCase);
+  }
+
+  findUseCase(message: TelegramMessage): UseCase {
```

Comment:
findAndTriggerUseCase(message: TelegramMessage): UseCase{

Reply:
Der Use Case wird nicht direkt getriggert sondern nur zurückgegeben, ichhabe das jetzt in findUseCaseByTrigger umbenannt. 

Es gibt nämlich auch noch die Möglichkeit, einen Use Case über den Namen zu finden, wird bei proaktivem Aufruf verwendet:
https://github.com/JaydenUlysses/Olaf-the-coolest-of-Bots/blob/86a9f29cbbb60f242bec0ecb73cd9bb8e39e4801/src/core/messageRouter.ts#L28